module/outputs.tf

```diff
@@ -29,3 +29,27 @@ output "step_function_get_messages_arn" {
 output "step_function_send_messages_arn" {
   value = aws_sfn_state_machine.send_message.arn
 }
+
+output "lambda_check_send_parameters_log_group_name" {
+  value = aws_cloudwatch_log_group.check_send_parameters.name
+}
+
+output "lambda_fetch_message_chunk_log_group_name" {
+  value = aws_cloudwatch_log_group.fetch_message_chunk.name
+}
+
+output "lambda_poll_mailbox_log_group_name" {
+  value = aws_cloudwatch_log_group.poll_mailbox.name
+}
+
+output "lambda_send_message_chunk_log_group_name" {
+  value = aws_cloudwatch_log_group.send_message_chunk.name
+}
+
+output "stepfunction_get_messages_log_group_name" {
```

Comment:
I think move these two adjacent to their outputs above for the state machine - just so they are grouped around the resource.